### ISSUE TITLE
Replace simple-glob for glob

### ISSUE THREAD

**Issue by f3l1x** [nwjs-community/nw-builder#489]
I'm getting this warning during npm install.

```
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated minimatch@0.3.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
```

It's because nw-builder is using simple-glob, which is quite old.

```
├─┬ nw-builder@3.5.1
│ └─┬ simple-glob@0.1.0
│   ├─┬ glob@3.2.11
│   │ └── minimatch@0.3.0 
│   └── minimatch@0.2.14 
```

I can prepare PR to depends on glob@6. 
Do you agree?

**Comment by zaygraveyard** [comment]
I agree but I'm not a maintainer we're gonna have to wait for @adam-lynch

**Comment by f3l1x** [comment]
Yep, let me known later.

**Comment by piwonesien** [comment]
I've updated the simple-glob to version 0.2.0 and fix this in #484. The warnings should disappear if this pr would be merged.

**Comment by kessler** [comment]
merged and published to npm
